Matter_Engine/Query/MaterialComposition.cs

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Dimensional;
+using BH.oM.Geometry;
+using BH.oM.Physical.Materials;
+using BH.oM.Quantities.Attributes;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.Engine.Matter
+{
+    public static partial class Query
+    {
+        /******************************************/
+        /****            IElement1D            ****/
+        /******************************************/
+
+        [Description("Gets all the Material's a elements is composed of and in which ratios")]
+        [Input("elementM", "The element to get the materialcomposition of")]
+        [Output("materialComposition", "The kind of matter the element is composed of and in which ratios")]
```

Comment:
```suggestion
        [Description("Gets the relative proportions of unique Materials an element is composed.")]
        [Input("elementM", "The element to get the MaterialComposition from.")]
        [Output("materialComposition", "The kind of matter the element is composed of and in which ratios")]
```

Reply:
```suggestion
        [Description("Gets the relative proportions of unique Materials of which an element is composed.")]
        [Input("elementM", "The element to get the MaterialComposition from.")]
        [Output("materialComposition", "The kind of matter the element is composed of and in which ratios")]
```

Didn't sound completely right, but aren't sure, could I get a double take? @al-fisher

Reply:
Yeah - fair. Will improve readability! Thanks @kThorsager

Reply:
How about:

Gets the unique Materials along with their relative proportions defining an object's make-up.

Reply:
@kThorsager  see above ^^ 😄